docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -46,4 +73,34 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold` (Optional):: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.4` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+[float]
+[[dfanalytics-dest-resources]]
+==== Dest configuration objects
+
+The `dest` configuration object has the following properties:
+
+`index` (Required)::
+  (string) The name of the index in which to store the results of the 
+  {dfanalytics-job}.
+
+`results_field` (Optional)::
+  (string) The name of the field in which to store the results of the analysis. 
+  The default value is `ml`.
+  
+[float]
+[[dfanalytics-source-resources]]
+==== Source configuration objects
+
+The `source` configuration object has the following properties:
+
+`index` (Required)::
+  (string) The name of the index on which to perform the analysis.
```

Comment:
This could be an array of strings. Similar to `datafeed.indices` though here we use the term `index`. We should probably explain this could be a single index or pattern, or an array of indices/patterns.